index.d.ts

```diff
@@ -103,7 +102,7 @@ export interface ImportDirective extends BaseASTNode {
   type: 'ImportDirective';
   path: string;
   unitAlias: string,
-  symbolAliases: [string, string][];
+  symbolAliases: Array<[string, string]>;
```

Comment:
This was another thing caught by the linter. Can you double check this is correct, please?